x-pack/test/functional_with_es_ssl/apps/cases/group1/sub_privileges.ts

```diff
@@ -119,6 +128,59 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
         });
       }
     });
+
+    describe('cases_settings', () => {
+      beforeEach(async () => {
+        await cases.api.createNthRandomCases(2);
```

Comment:
nit: but is this even necessary? We are navigating to settings. (`afterEach` also)